applications/CompressiblePotentialFlowApplication/tests/test_CompressiblePotentialFlowApplication.py

```diff
@@ -32,7 +32,7 @@ def AssembleTestSuites():
     # - testSmallExample
     smallSuite = suites['small']
     smallSuite.addTest(PotentialFlowTests('test_Naca0012SmallAdjoint'))
-    #smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TestPotentialElement2D]))
+    smallSuite.addTest(PotentialFlowTests('test_EmbeddedCircleNoWake'))
```

Comment:
check e.g. [here](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py#L389) how to automatically add all tests from a test case, this way you will never forget to add it

=> I see you had this already (commented), I highly recommend using it. A test that is not run is kinda useless ;)

Reply:
Can you elaborate what does that line do? Right now if i run the script `test_CompressiblePotentialFlowApplication.py` the 'Naca0012' test and 'Embedded' test run correctly, along the cpp_tests.

Reply:
it detects all tests that are in the test-class => meaning that you don't have to add every single test manually

imagine you write a new test, e.g. `test_EmbeddedCircleNoWake_2`, but you forget to add it to the smallsuite, then it will never be executed
This won't happen if you use `loadTestsFromTestCases`, which does what I explained

Reply:
Doing the following (note that the test addition is commented):
```
    smallSuite = suites['small']
    # smallSuite.addTest(PotentialFlowTests('test_Naca0012SmallAdjoint'))
    # smallSuite.addTest(PotentialFlowTests('test_EmbeddedCircleNoWake'))
    KratosUnittest.TestLoader().loadTestsFromTestCases([PotentialFlowTests])
```

Seems to do nothing (no tests are run). Am I required to add every specific test regardless?

Reply:
yes, you are not adding the tests to the smallsuite
=> `smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([PotentialFlowTestsement]))` should work

Reply:
Ok now it works! Thanks!

Reply:
if in doubt check how it is done in StructuralMechanics, there it is used exensively